test/unit/plugins/hosts/darwin/cap/rdp_test.rb

```diff
@@ -18,13 +18,19 @@
     expect(result).to match("full address:s:host:port")
     expect(result).to match("prompt for credentials:i:1")
     expect(result).to match("username:s:username")
+    result.lines.each do |line|
+      expect(line).to match(/\r\n$/)
+    end
```

Comment:
This test asserts that one or more lines in the file end in `\r\n`, which isn't the same guarantee that each setting appears on it's own line. The matches should be re-written as array inclusion tests against the content of the file:

```ruby
result = File.readlines(path).map(&:chomp)
expect(result).to include("drivestoredirect:s:*")
expect(result).to include("full address:s:host:port")
expect(result).to include("prompt for credentials:i:1")
expect(result).to include("username:s:username")
```